dev/archery/archery/lang/python.py

```diff
@@ -139,10 +143,19 @@ def traverse(self, fn, obj, from_package):
                     continue
 
                 member = getattr(obj, name)
-                module = getattr(member, '__module__', None)
-                if not (module and module.startswith(from_package)):
+                module = _get_module(member)
+                if module is None or not module.startswith(from_package):
                     continue
-
+                # Is it a Cython-generated method? If so, try to detect
+                # whether it has a implicitly-generated docstring due to
+                # the Cython `embedsignature` directive; and if so, skip
```

Comment:
`inspect_signature` supports cython's signature, so we shouldn't skip those symbols.

Reply:
The problem is not the signature, it's that the generated docstring doesn't have parameter description, so this would fail the numpydoc rule PR01.

Reply:
Yes, exactly. I could improve the comment.